docs/model_developers/modules.rst

```diff
@@ -329,3 +329,61 @@ for supplying the agents with the appropriate ``inmanta_plugins`` packages.
 The only exception to this rule is when using the ``inmanta export`` command. It exports a project and all its modules'
 ``inmanta_plugins`` packages to the orchestrator server. When this method is used, the orchestrator does not install any modules
 from the Python package repository but instead contains all Python code as present in the local Python environment.
+
+Configure a project to install modules from a private python package repository
```

Comment:
We don't really configure the project here. The configuration of the project is independent from whether or not authentication is required on the remote Python package repository. It's the configuration of the server that has to be updated.
```suggestion
Configure the Inmanta server to install modules from a private python package repository
```